compiler/rustc_driver/src/lib.rs

```diff
@@ -1323,9 +1323,54 @@ pub fn init_env_logger(env: &str) {
     tracing::subscriber::set_global_default(subscriber).unwrap();
 }
 
+extern "C" {
+    // Only available in glibc
```

Comment:
nit: Not sure if this is true, it seems available on macOS which I don't think uses glibc?